### DISCUSSION TITLE
Error building Windows application

### DISCUSSION THREAD

**Discussion by RectNP** [flxzt/rnote#366] [General]
Hello.

When i try to build the application i get the error:
meson.build:74:0: ERROR: Program 'cmake' not found or not executable

[log here](https://pastebin.com/V1UPnmmA)

**Comment by flxzt** [comment]
Hey, the windows build instructions are slightly out of date and are missing some dependencies such as cmake. my efforts fixing the build and up to date documentation are in the [windows installer](https://github.com/flxzt/rnote/tree/windows_installer) branch. I couldn't resolve all the errors myself though, I am getting some linker errors with the pthread library. So the windows build is kind of broken at the moment.

**Reply by RectNP** [reply]
Thanks for your reply.
